.github/SUPPORT.md

```diff
@@ -7,26 +7,32 @@ Take a look at those mediums listed at <https://vector.dev/community>
 
 ## How to ask a question about Vector
 
-Whether in our community [Discord][discord] server or [GitHub Discussions][discussions], framing
-your question well and providing the right level of details will improve your chances of getting
+Framing your question well and providing the right level of details will improve your chances of getting
 your question answered. Here are some tips:
 
 ### Before asking
 
-Check the [Vector documentation](https://vector.dev/docs/) first to see if it answers your question.
+Check the [Vector documentation](https://vector.dev/docs/) first to see if it answers your question. If you use a search
+engine
+and provide a query like the following
+`site:vector.dev releases`, you might get better results than the built-in search.
```

Comment:
This is true, but probably means we should improve the site search 😅 Maybe some feedback to propagate to the website team.